sts/internal/types.go

```diff
@@ -0,0 +1,590 @@
+/*
+Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+// The sts/internal package provides the types for invoking the sts.Issue method.
+// The sts.Issue and SessionManager LoginByToken methods require an XML signature.
+// Unlike the JRE and .NET runtimes, the Go stdlib does not support XML signing.
+// We should considering contributing to the goxmldsig package and gosaml2 to meet
+// the needs of sts.Issue rather than maintaining this package long term.
```

Comment:
> We should considering contributing to the goxmldsig package and gosaml2 to meet the needs of sts.Issue rather than maintaining this package long term.

:+1: